huaweicloud/services/acceptance/coc/resource_huaweicloud_coc_war_room_test.go

```diff
@@ -0,0 +1,101 @@
+	}
+
+	return coc.GetWarRoom(client, state.Primary.ID)
+}
+
+func TestAccResourceWarRoom_basic(t *testing.T) {
+	var obj interface{}
+	rName := acceptance.RandomAccResourceName()
+	resourceName := "huaweicloud_coc_war_room.test"
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&obj,
+		getWarRoomResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckCocApplicationID(t)
+			acceptance.TestAccPreCheckUserId(t)
+			acceptance.TestAccPreCheckCocRoleID(t)
+			acceptance.TestAccPreCheckCocSceneID(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {
+				Source:            "hashicorp/time",
+				VersionConstraint: "0.12.1",
+			},
+		},
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testWarRoom_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+					resource.TestCheckResourceAttrSet(resourceName, "war_room_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "title"),
+					resource.TestCheckResourceAttrSet(resourceName, "admin"),
+					resource.TestCheckResourceAttrSet(resourceName, "incident.#"),
+					resource.TestCheckResourceAttrSet(resourceName, "incident.0.id"),
+					resource.TestCheckResourceAttrSet(resourceName, "incident.0.incident_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "incident.0.is_change_event"),
+					resource.TestCheckResourceAttrSet(resourceName, "incident.0.failure_level"),
+					resource.TestCheckResourceAttrSet(resourceName, "occur_time"),
+					resource.TestCheckResourceAttrSet(resourceName, "create_time"),
+					resource.TestCheckResourceAttrSet(resourceName, "circular_level"),
+					resource.TestCheckResourceAttrSet(resourceName, "war_room_status.#"),
+					resource.TestCheckResourceAttrSet(resourceName, "war_room_status.0.id"),
+					resource.TestCheckResourceAttrSet(resourceName, "war_room_status.0.name_zh"),
+					resource.TestCheckResourceAttrSet(resourceName, "war_room_status.0.name_en"),
+					resource.TestCheckResourceAttrSet(resourceName, "war_room_status.0.type"),
+					resource.TestCheckResourceAttrSet(resourceName, "impacted_application.#"),
+					resource.TestCheckResourceAttrSet(resourceName, "impacted_application.0.id"),
+					resource.TestCheckResourceAttrSet(resourceName, "enterprise_project_id"),
+				),
+			},
+		},
+	})
+}
+
+func testWarRoom_basic(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "huaweicloud_coc_war_room" "test" {
+  war_room_name         = "%[2]s"
+  application_id_list   = ["%[3]s"]
+  incident_number       = huaweicloud_coc_incident.test.id
+  war_room_admin        = "%[6]s"
+  enterprise_project_id = "0"
+  schedule_group  {
+    role_id             = "%[4]s"
+    scene_id            = "%[5]s"
+  }
```

Comment:
suggest to sort the params for func
```suggestion
%[1]s

resource "huaweicloud_coc_war_room" "test" {
  war_room_name         = "%[2]s"
  application_id_list   = ["%[3]s"]
  incident_number       = huaweicloud_coc_incident.test.id
  war_room_admin        = "%[4]s"
  enterprise_project_id = "0"
  
  schedule_group {
    role_id  = "%[5]s"
    scene_id = "%[6]s"
  }
```